tensorflow/core/framework/tensor.cc

```diff
@@ -563,6 +563,14 @@ struct ProtoHelper<float8_e5m2> : public Float8ProtoHelper<float8_e5m2> {};
 template <>
 struct ProtoHelper<float8_e4m3fn> : public Float8ProtoHelper<float8_e4m3fn> {};
 
+template <>
```

Comment:
Nit: Format the code the same way as that for ocp fp8?

Reply:
I believe it's auto-formatted by `clang-format`.
https://github.com/ROCm/tensorflow-upstream/blob/develop-upstream/tensorflow/.clang-format
![image](https://github.com/user-attachments/assets/58e6a51f-0732-4580-81d2-731c1672288e)